modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,143 @@
+====
+
+* With this update, two new fields `.spec.applicationSet.sourceNamespaces` and `.spec.applicationSet.scmProviders` are introduced in the `ArgoCD` CRD to support `ApplicationSet` in any namespaces. Administrators can use these fields to define certain namespaces that manage `ApplicationSet` resources. link:https://issues.redhat.com/browse/GITOPS-3754[GITOPS-3754]
+
+* With this update, Argo CD server has the required permissions to manage `ApplicationSet` resources. link:https://issues.redhat.com/browse/GITOPS-3762[GITOPS-3762]
+
+* With this update, you can configure log levels, such as `debug`, `info`, `warn`, `error`, `panic` and `fatal`. The default log level set for the output is `info`. To change the log level, add the environment variable `LOG_LEVEL` in the `.spec.config.env` field of the `Subscription` CR. link:https://issues.redhat.com/browse/GITOPS-4016[GITOPS-4016]
++
+.Example output
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  ...
+  config:
+    env:
+    - name: LOG_LEVEL
+      value: "error"
+----
+
+* With this update, Argo CD accepts the wildcard values in the `sourceNamespaces` field so that you can specify multiple namespaces or patterns of namespaces. To use this feature, specify the namespaces where Argo CD can manage applications in the `.spec.sourceNamespaces` field of the `ArgoCD` CR. link:https://issues.redhat.com/browse/GITOPS-3935[GITOPS-3935]
++
+.Example
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd-wildcard-pattern
+spec:
+  sourceNamespaces:
+    - app-team-*
+   - namespace-2
+----
+
+In the previous example, permissions are granted to namespaces that match the pattern `app-team-*`, such as `app-team-1`, `app-team-2`, and namespace-2, which does not use wildcard values.
+
+To grant permissions for all the namespaces on the Argo CD cluster using the `*` wildcard pattern, configure the `ArgoCD` CR in the following manner:
++
+.Example
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd-all-namespaces
+spec:
+  sourceNamespaces:
+    - '*'
+----
+
+[id="fixed-issues-1-12-0_{context}"]
+== Fixed issues
+
+* Before this update, the notifications controller could not use built-in functions in notification templates to obtain information about applications due to an incorrectrepository server address. This update fixes the issue by initializing the notification controller to use the correct repository server address. link:https://issues.redhat.com/browse/GITOPS-2867[GITOPS-2867]
```

Comment:
```suggestion
* Before this update, the notifications controller could not use built-in functions in notification templates to obtain information about applications because of an incorrect repository server address. This update fixes the issue by initializing the notification controller to use the correct repository server address. link:https://issues.redhat.com/browse/GITOPS-2867[GITOPS-2867]
```

Reply:
Fixed.